paper/myst.yml

```diff
@@ -2,48 +2,58 @@
 version: 1
 project:
   id: ubcgif-2020-oldenburg-etal-exploration-geophysics-open-source
-  # title: TITLE OF PAPER
-  # description: SHORT DESCRIPTION HERE
+  title: 3D electromagnetic modelling and inversion - a case for open source
+  description: Presenting arguments for adopting an open-source methodology for geophysics and provide some background about open-source software for electromagnetics.
   authors:
-    # - name: AUTHOR 1
-    #   email: EMAIL
-    #   corresponding: true
-    #   orcid: ORCID
-    #   affiliation:
-    #     id: ubc
-    #     name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
-    # - name: Lindsey J. Heagy
-    #   email: lheagy@eoas.ubc.ca
-    #   corresponding: true
-    #   orcid: 0000-0002-1551-5926
-    #   affiliation: ubc
-  # doi: DOI
-  # arxiv: ARXIV
-  # github: GITHUB-URL
-  # date: YYYY/MM/DD # date of publication
+    - name: Douglas W. Oldenburg
+      email: doug@eos.ubc.ca
+      corresponding: true
+      orcid: 0000-0002-4327-2124
+      affiliation:
+        id: ubc
+        name: Department of Earth, Ocean and Atmospheric Sciences, University of British Columbia, Vancouver, British Columbia
+    - name: Lindsey J. Heagy
+      email: lheagy@eoas.ubc.ca
+      orcid: 0000-0002-1551-5926
+      affiliation: ubc
+    - name: Seogi Kang
+      affiliation: Department of Geophysics, Stanford University, Stanford, California
```

Comment:
For the affiliations, lets use UBC for everyone here since that is where we all were at the time, so both Seogi and Rowan can be updated to have a ubc affiliation 
![image](https://github.com/ubcgif/2020-oldenburg-etal-exploration-geophysics-open-source/assets/6361812/490d75f9-bc4b-4694-adca-6c865e3e18a3)

Reply:
for this one i just copied their information from [2024-heagy-etal-tle-future-of-applied-geophysics](https://github.com/ubcgif/2024-heagy-etal-tle-future-of-applied-geophysics). Should we change there as well?